bats/fb-test-foreman.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+@test "check web app is up" {
+  curl -sk "https://localhost$URL_PREFIX/users/login" | grep -q login-form
+}
+
+@test "wake up puppet agent" {
+  source ~/.bashrc
+  puppet agent -t -v
+}
+
+@test "check web app is still up" {
+  curl -sk "https://localhost$URL_PREFIX/users/login" | grep -q login-form
+}
+
+@test "check smart proxy is registered" {
+  hammer --csv proxy list | grep -q "$(hostname -f)"
+}
+
+@test "check host is registered" {
+  hammer host info --name "$(hostname -f)"
+}
+
+@test "check hammer ping" {
+  hammer ping
+}
```

Comment:
I'm **guessing** Foreman Bats, but no real preference from my side. I'm not even sure if we should continue with writing tests in bats. Personally I'm still interested in getting a Robotello test set that is light enough to quickly test, but I haven't used it yet so I don't know if that's feasible.